### PR TITLE
6 dev

### PR DESCRIPTION
**Pull request by DoYouEvenCpp**
Fixes: #6

### CODE REVIEW

File: source/exec/builtin_function_caller.hpp [hunk 1]
```diff
@@ -150,6 +150,8 @@ class builtin_function_caller
                                const params_t& params);
   inst::instance* string_size(inst::instance& instance,
                               const params_t& params);
+  inst::instance* string_operator_equal(inst::instance& instance,
+                                              const params_t& params);
```
Comment (stryku) [inline source/exec/builtin_function_caller.hpp]: Please format the file.

Comment (DoYouEvenCpp) [reply source/exec/builtin_function_caller.hpp]: Sure, fixed.

**Comment by stryku** [comment]
Please rebase and squash commits

**Comment by DoYouEvenCpp** [comment]
Will post a new PR for

**Pull request closed without merging**